wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -115,25 +114,40 @@ def parse_article(self, response):
         """ Scrape the article metadata from the detailed article page. Then,
         redirect to the PDF page.
 
-        @url http://apps.who.int/iris/handle/10665/123400
+        @url http://apps.who.int/iris/handle/10665/272346?show=full
         @returns requests 1 1
         @returns items 0 0
         """
 
-        year = response.meta.get('year', {})
         data_dict = {
-            'Year': year,
+            'year': response.meta.get('year', {}),
         }
-        for tr in response.css('table.itemDisplayTable tr'):
-            label = tr.css('td.metadataFieldLabel::text').extract_first()
-            label = label[:label.find(':')]
-            # Remove HTML markdown for some metadata are in a <a>
-            value = clean_html(tr.css('td.metadataFieldValue').extract_first())
+        data_dict['title'] = response.css(
+            'h2.page-header::text'
+        ).extract_first()
 
-            data_dict[label] = value
+        details_dict = defaultdict(list)
+        for line in response.css('.detailtable tr'):
+
+            # Each tr should have 2 to 3 td: attribute, value and language.
```

Comment:
What if it has more than 3?

Reply:
We're only interested in the first two columns, so if there are more, we can just ignore it (e.g. the third column, when it has one, is the language of the attribute's value).

Reply:
There should be a command in the code to this effect.